player/lua/osc.lua

```diff
@@ -2072,6 +2103,16 @@ local function osc_init()
     bind_mouse_buttons("volume")
 
 
+    -- custom buttons
+    for i = 1, 5 do
```

Comment:
Can this be made more generic and allow N buttons and stop once no more is provided by user?

Reply:
Changed to 99. The problem with passing `--script-opts` that were not defined is that it logs a warning.

Also maybe we should ship the whole fluent system icons font and make this configurable with its icon names instead of emoji since emoji can be misaligned, if we can understand how to do that.